src/tabs/tabs.js

```diff
@@ -6,6 +6,10 @@ angular.module('ui.bootstrap.tabs', [])
   ctrl.tabs = [];
 
   ctrl.select = function(index, evt) {
+    if (evt && evt.which === 2) {
```

Comment:
So `evt.which` is jQuery normalization - this should still be fine, but somewhat unreliable since this needs to be detected via `mousedown`/`mouseup` events.